### DISCUSSION TITLE
Can Maddy show the specific TLS and cipher suite information of the incoming letter?

### DISCUSSION THREAD

**Discussion by cherinyy** [emersion/maddy#542] [Q&A]
For example Gmail does this:

```
Received: from ...
        by mx.google.com with ESMTPS id ...
        for <...>
        (version=TLS1_2 cipher=ECDHE-ECDSA-AES128-GCM-SHA256 bits=128/128);
        ...
```

**Comment by foxcpp** [comment]
Not implemented currently. Should be easy to add, but I wonder why exactly you want to have this information?

**Reply by cherinyy** [reply]
Maybe it's just a habit, but from time to time I check the encryption suite that my browser uses to access the site.
